src/guide/qtcreator.md

```diff
@@ -11,6 +13,8 @@ Install [Qt Creator version 4.0 or above](https://www.qt.io/ide/).
 ```
 # On Fedora 22 and up
 sudo dnf install qtcreator
+# On Ubuntu 16.04
+sudo apt-get install qtcreator
```

Comment:
I think since 16.04 you could also use `apt`: https://www.maketecheasier.com/apt-vs-apt-get-ubuntu/